src/ontology/mondo-ingest.Makefile

```diff
@@ -182,6 +182,22 @@ config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
+# TODO:
+# $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_in_mondo.tsv: $(TMPDIR)/mondo.sssom.tsv
+# TODO: temp for when I don't want `python-install-dependencies`
+$(REPORTDIR)/excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/excluded_terms_in_mondo.tsv: $(TMPDIR)/mondo.sssom.tsv tmp/mondo.owl python-install-dependencies
+	python3 ../analysis/excluded_terms_in_mondo.py \
+	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
```

Comment:
@matentzn Which style do you prefer?
a. Named, e.g. `--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv`
b. Positional, e.g. `--mondo-mappings-path $(word 1,$^)`

I don't know why I took to these positional ones in my last PR. Maybe because it was new and fancy to me. I think we both prefer (a), but just wanted to clarify with you.

Reply:
I prefer (a) because of the risk of changing dependency order.. And its also more readable!

Reply:
sounds good